src/proxy.js

```diff
@@ -113,24 +76,23 @@ var RdHandler = {
   },
 
   /**
-   * Handler for incoming requests to Requests Debugger Tool proxy server.
-   * @param {http.IncomingMessage} clientRequest 
-   * @param {http.ServerResponse} clientResponse 
+   * Handler for incoming requests to Requests Debugger Tool server.
+   * @param {http.IncomingMessage} clientRequest
+   * @param {http.ServerResponse} clientResponse
    */
   requestHandler: function (clientRequest, clientResponse) {
     clientRequest.id = ++RdHandler._requestCounter + '::' + uuidv4();
-
+    var url = clientRequest.url;
```

Comment:
do we really need this? we are already extracting that in the below object....

Reply:
`request.url`...

Reply:
fixed

Reply:
can be resolved.